docs/source/glossary.rst

```diff
@@ -6,15 +6,52 @@ Glossary
 
 .. glossary::
 
-    simulant
+    Attribute
+        A variable associated with each :term:`simulant <Simulant>`. For
+        example, each simulant may have an attribute to describe their age or
+        position.
+
+    Component
+        Any self-contained piece of code that can be plugged into the
+        simulation to add some functionality. In ``vivarium`` we typically
+        think of components as encapsulating and managing some behavior or
+        attributes of the :term:`simulants <Simulant>`.
+
+    Configuration
+        The configuration is a a set of parameters a user can modify to
```

Comment:
Hmm is this definition accurate? Because configuration parameters affect more than just components (specifically, the time block affects the framework itself, not any component). Maybe that's more detail than we care about here.

Reply:
More detail than we care about.